js/cl.js

```diff
@@ -8,6 +8,7 @@ __                .__                        __
 ====================================================*/
 
 function codeLane(holder,width,height,data,custom_options){
+    
```

Comment:
Hello. I watched your video on embold YouTube, and was wondering, how did you get the Embold step to show up as a CI step on GitHub? The video suggests this would happen automatically after linking my GitHub repo, but it has not appeared. I can only see if my PR's have passed Quality Gate on the Embold Platform. I am wondering how you did this please?